packages/terra-tag/src/Tag.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Tag.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  TAB: 9,
+};
+
+const tagTheme = {
```

Comment:
is this meant to act as an enum for proptypes? If so, could we change the `theme` typecheck below to be 
```js
theme: PropTypes.oneOf(...Object.values(tagTheme))
```
? So long as your values in `tagTheme` are always strings, the typecheck on `theme` below being a string becomes implicit, you limit yourself to only choosing one of these options and you separate your valid types (enum) from your typechecking logic. 

I would also suggest in that case making this const

```js
const TagThemes = {
  NORMAL: 'normal',
};

...

Tag.Opts = {};
Tag.Opts.Themes = TagThemes;

export default Tag;
```

Doing this would allow the user to access the valid themes in their consumption of `Tag` by doing

```jsx
import Tag from '~/Tag';

const Foo = () => (
  <Tag 
    /* other props */
    theme={Tag.Opts.Themes.NORMAL}
  />
);
```

Reply:
As we discussed offline removed the theme prop, as there is only single theme for the current implementation.